packages/transactions/src.ts/index.ts

```diff
@@ -543,6 +549,8 @@ export async function parse(rawTransaction: BytesLike): Promise<Transaction> {
         contents.data = hexlify(Buffer.from(parsed.contents));
     } else if (parsed instanceof TransferTransaction) {
         // TODO populate value / to?
```

Comment:
We can populate `value` - I think that this will be the account's `initialBalance`.

Reply:
Resolved